django_rest_passwordreset/views.py

```diff
@@ -41,25 +41,31 @@ def post(self, request, *args, **kwargs):
         serializer = self.serializer_class(data=request.data)
         serializer.is_valid(raise_exception=True)
         token = serializer.validated_data['token']
-
+        response_dict = dict({"status_code": None, "message": None})
         # get token validation time
         password_reset_token_validation_time = get_password_reset_token_expiry_time()
 
         # find token
         reset_password_token = ResetPasswordToken.objects.filter(key=token).first()
 
         if reset_password_token is None:
-            return Response({'status': 'notfound'}, status=status.HTTP_404_NOT_FOUND)
+            message = get_response_message("TOKEN_NOT_FOUND")
+            response_dict.update({"status_code": 404, "message": message})
+            return Response(response_dict, status=status.HTTP_404_NOT_FOUND)
 
         # check expiry date
         expiry_date = reset_password_token.created_at + timedelta(hours=password_reset_token_validation_time)
 
         if timezone.now() > expiry_date:
             # delete expired token
             reset_password_token.delete()
-            return Response({'status': 'expired'}, status=status.HTTP_404_NOT_FOUND)
-        
-        return Response({'status': 'OK'})
+            message = get_response_message("TOEKN_EXPIRED")
```

Comment:
Please fix typo

Reply:
typo fixed TOEKN -> TOKEN

Reply:
re-introduced the "status" field in # https://github.com/anx-ckreuzberger/django-rest-passwordreset/pull/65/commits/d7429b68436b81ab65f95c22c767a795dc83d5cc